pages/faq.tsx

```diff
@@ -1,13 +1,81 @@
 import { Accordion, AccordionSection } from "../components/Accordion";
-
-import { ExternalLink } from "../components/ExternalLink";
 import { Flex } from "../components/flex/Flex";
 import Layout from "../components/layout/Layout";
 import METADATA from "../constants/metadata";
 import { PageRoot } from "../components/layout/PageRoot";
 import React from "react";
+import { fetchFAQItems } from "../services/cms";
+import { documentToReactComponents } from "@contentful/rich-text-react-renderer";
+import { INLINES } from "@contentful/rich-text-types";
+import styled from "styled-components";
+
+const StyledLink = styled.a`
+  text-decoration: underline;
+`;
+
+export const getStaticProps: any = async () => {
+  const { entries: _entries, total } = await fetchFAQItems();
+  const entries: any = {};
+  _entries.forEach((entry: any) => {
+    if (!entries[entry.tag]) {
+      entries[entry.tag] = [];
+    }
+    entries[entry.tag].push(entry);
+  });
+
+  return {
+    props: {
+      entries,
+      total,
+    },
+    revalidate: 100,
+  };
+};
+
+export default function Faq(props: any) {
+  const { entries: faqItems } = props;
+
+  const docToReactOption = {
+    renderNode: {
+      [INLINES.HYPERLINK]: ({ data }, children) => (
```

Comment:
You will need to add `// eslint-disable-next-line react/display-name` here because my IDEA displayed an error. Styled Components can cause linting issues sometimes. See the issue here https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/display-name.md